lib/node/rpc.js

```diff
@@ -2477,6 +2478,27 @@ class RPC extends RPCBase {
     return proof.hash().toString('hex');
   }
 
+  async validateResource(args, help) {
+    if (help || args.length < 1 || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'validateresource'
+        + '"{"records": [{...}]}"');
```

Comment:
nit: this doesn't print out correctly.
Current:
```
$ hsd-rpc --network=regtest validateresource
validateresource"{"records": [{...}]}"
```

Suggestion:
```
$ hsd-rpc --network=regtest validateresource
validateresource '{"records": [{...}]}'
```

Note the single quotes as well, you may have to use `\'` in the code.

Reply:
Good find! Pushed up a fix.